release notes/v0.5.0.md

```diff
@@ -0,0 +1,100 @@
+xk6-browser v0.5.0 is here! :tada:
+
+This minor release contains a few important stability fixes, some breaking changes, improvements, and a continuation of our efforts to migrate to async APIs.
+
+
+## Bugs fixed
+
+- A couple of `Page.goto()` race conditions, where the call would timeout waiting for the navigation event. This was mainly found to occur in CI, but it could happen in real world tests as well.
+  ([#480](https://github.com/grafana/xk6-browser/pull/480), [#501](https://github.com/grafana/xk6-browser/pull/501))
+
+- A `Page.waitForNavigation()` race condition, where the call would fail with a Go nil pointer dereference error. ([#500](https://github.com/grafana/xk6-browser/pull/500))
+
+- Combining keys presses when inputting text in an input field now works with the `Shift` modifier key ([#326](https://github.com/grafana/xk6-browser/pull/326), [#522](https://github.com/grafana/xk6-browser/pull/522)).
```

Comment:
```suggestion
- Combining key presses when inputting text in an input field now works with the `Shift` modifier key ([#326](https://github.com/grafana/xk6-browser/pull/326), [#522](https://github.com/grafana/xk6-browser/pull/522)).
```

Though isn't this more of a new feature instead of a bug fix? #285 was labeled as `enhancement`, not `bug`.

Reply:
Good point. I've moved it into `New Feature` in commit 7ce1ac235422686146bed1f1d31496899d93959b.